compliance_checker/cf/cf.py

```diff
@@ -2513,7 +2516,7 @@ def check_cell_boundaries(self, ds):
             if boundary_variable_name not in ds.variables:
                 valid = False
                 reasoning.append("Boundary variable {} referenced by {} not "
-                                 "found in dataset variables".format(boundary_variable.name,
+                                 "found in dataset variables".format(boundary_variable_name,
```

Comment:
`boundary_variable.name` does not exists and this never failed, so I am guessing this if-clause is never True.
I changed it to `boundary_variable_name` because that one exists but I am not sure that is correct!